src/Transfers/ImportRequestModel.cs

```diff
@@ -28,6 +28,10 @@ public class ImportRequestModel
         /// </summary>
         public bool WaitForCompletion { get; set; } = false;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether notifications are sent.
+        /// </summary>
+        public bool SendNotification { get; set; } = true;
         /// <summary>
```

Comment:
Add a blank line below the statement for readability

Reply:
will do hmm